zproject/dev_settings.py

```diff
@@ -21,4 +21,5 @@
 # Disable Camo in development
 CAMO_URI = ''
 OPEN_REALM_CREATION = True
+TERMS_OF_SERVICE = 'zproject/terms.md.template'
```

Comment:
Do we want to install this template in production systems, like we do with local_settings_template.py?